### ISSUE TITLE
android 9.0  加载时状态栏会消失

### ISSUE THREAD

**Issue by chinahbpyj** [Kaopiz/KProgressHUD#56]
android 9.0系统，一加5，loading时状态栏会消失，加载完毕会显示出来

**Comment by tuanna-hsp** [comment]
Did you confirm that issue with any other Android 9 devices or emulator? It seems fine in my case.

**Comment by chinahbpyj** [comment]
Hello, I just tried it with mi 8. Android 9.0 system still has the same problem

**Comment by tuanna-hsp** [comment]
There may be a conflict between your project theming style and the HUD dialog.
Though I don't know for sure without some context.
Can you download the demo project and confirm if the problem also happens?

**Comment by chinahbpyj** [comment]
Yes, I changed your source code according to the way you said. I just added the style when I created the ProgressDialog. You can provide the added style
